client/db/bolt/db_test.go

```diff
@@ -224,6 +224,70 @@ func TestAccounts(t *testing.T) {
 	}
 }
 
+func TestDeleteAccount(t *testing.T) {
+	boltdb := newTestDB(t)
+	acct := dbtest.RandomAccountInfo()
+	host := acct.Host
+	boltdb.CreateAccount(acct)
+
+	err := boltdb.deleteAccount(acct)
+
+	if err != nil {
+		t.Fatalf("Unexpected deleteAccount error: %v", err)
+	}
+	actualAccount, err := boltdb.Account(host)
+	if err == nil {
+		t.Fatalf("Expected account not found error")
+	}
+	if actualAccount != nil {
+		t.Fatalf("Did not expect to retrieve deleted account.")
+	}
+}
+
+func TestDisableAccount(t *testing.T) {
+
+	boltdb := newTestDB(t)
+	acct := dbtest.RandomAccountInfo()
+	host := acct.Host
+	boltdb.CreateAccount(acct)
+
```

Comment:
Maybe make sure this has not errored, And also make sure `boltdb.DisabledAccount(acct.EncKey)` does error here.

Reply:
Fixed for next push.